azure-pipelines.yml

```diff
@@ -0,0 +1,18 @@
+trigger:
+- master
+- dev
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.5' 
```

Comment:
Might need to consider other python versions, but this is fine for now

Reply:
Yes, it's on my radar, but not my top priority